src/transcriber/main.py

```diff
@@ -0,0 +1,133 @@
+# /// script
+# dependencies = [
+#   "click",
+#   "faster-whisper",
+#   "jinja2",
+#   "tqdm",
+# ]
```

Comment:
I really don't think it's a good idea for us to do this. Recreating two different versions of the dependencies and how we run it and that is definitely not good